api/config/config_prod.json

```diff
@@ -101,7 +101,8 @@
     "enableBetaAccess": false,
     "unsafeAllowSelfBypass": false,
     "unsafeAllowUserCreationFromGSuiteData": false,
-    "enableAccessRenewal": false
+    "enableAccessRenewal": false,
```

Comment:
FYI for merge conflict awareness https://github.com/all-of-us/workbench/pull/5131